releasenotes/notes/29130.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+issue:
+  - 29125
+releaseNotes:
+  - |
+    **Improved** istio-agent will send proxy metadata specified in the mesh config file to istiod.
```

Comment:
Is this something the user needs to be aware of ? AFAIK this is internal detail, no behavior change.